timely_beliefs/visualization/utils.py

```diff
@@ -51,7 +53,18 @@ def plot(
         intuitive_forecast_horizon=intuitive_forecast_horizon,
     )
     unique_belief_horizons = plottable_df["belief_horizon"].unique()
-    event_value_range = (bdf.min()[0], bdf.max()[0])
+
+    # Set range of event values
+    if None in event_value_range:
+        event_value_range = list(event_value_range)
+        if event_value_range[0] is None:
+            # Infer minimum from data
+            event_value_range[0] = bdf["event_value"].min()
+        if event_value_range[-1] is None:
+            # Infer maximum from data
+            event_value_range[-1] = bdf["event_value"].max()
+        event_value_range = tuple(event_value_range)
+
```

Comment:
Maybe I'm overlooking something, but `event_value_range` doesn't seem to actually be applied anywhere?

Reply:
It is passed to `graphs.value_vs_time_chart` in line 113 (previously line 100).